data/json/items/comestibles/mutagen.json

```diff
@@ -20,75 +20,158 @@
         "flags" : [ "MUTAGEN_STRONG" ]
     },
     {
-        "id" : "mutagen_beast",
+        "abstract" : "mutagen_flavor", "//" : "meta item, not spawned in game but used as a baseline for class of mutagens",
```

Comment:
@mugling Besides the abstract part is the description fine? or should I just delete it.